client.go

```diff
@@ -0,0 +1,63 @@
+package nomadclient
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	dphttp "github.com/ONSdigital/dp-net/http"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type Nomad struct {
+	Client dphttp.Clienter
+	URL string
+	// Name needed for checker later.
+	// Name string
+}
+
+func NewClient(NomadEndpoint, NomadCACert string, NomadTLSSkipVerify bool) (*Nomad, error){
+
+	dpHTTPClient := *dphttp.DefaultClient
```

Comment:
Does this need to be `*dphttp.DefaultClient{}`?

Reply:
No that wouldn't work, if Im not mistaken I think you are thinking of instantiating an empty client using `&dhttp.Client{}` . Arguably we probably should call a function to get default client instead of creating a pointer to the default client directly, but this may not exist.